clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/ReservationReportCachingAgent.groovy

```diff
@@ -253,6 +287,13 @@ class ReservationReportCachingAgent implements CachingAgent {
     }
   }
 
+  private Cache getCacheView() {
```

Comment:
`AmazonLoadBalancerInstanceStateCachingAgent` is also doing this to late bind to `Cache`.